### PR TITLE
refactor color pallet

### PR DESCRIPTION
**Pull request by rezaerami**
- add new color pallet
- refactor old color pallet and use new ones
- add hex to rgba helper to use the same color pallet for colors with alpha value

### CODE REVIEW

File: packages/libs/react-ui/src/styles/colors.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  const $green = parseInt(hex.substring(2, 4), 16);
+  const $blue = parseInt(hex.substring(4, 6), 16);
+
+  opacity = Math.min(Math.max(opacity, 0), 1);
+
+  return `rgba(${$red}, ${$green}, ${$blue}, ${opacity})`;
+};
+
+// eslint-disable-next-line @kadena-dev/typedef-var
+export const colorPallet = {
```
Comment (eileenmguo) [inline packages/libs/react-ui/src/styles/colors.ts:13]:
```suggestion
export const colorPalett = {
```

File: packages/libs/react-ui/src/styles/vars.css.ts [hunk 2]
```diff
@@ -90,126 +91,126 @@ export const vars = createGlobalTheme(':root', {
...
+    $secondaryHighContrast: colorPallet.$pink100,
+    $secondaryAccent: colorPallet.$pink60,
+
+    $tertiarySurface: colorPallet.$purple40,
+    $tertiaryContrast: colorPallet.$purple80,
+    $tertiaryHighContrast: colorPallet.$purple100,
+    $tertiaryAccent: colorPallet.$purple60,
+
+    $borderDefault: hexToRgba(colorPallet.$gray50, 0.2),
+    $borderSubtle: hexToRgba(colorPallet.$gray40, 0.4),
```
Comment (eileenmguo) [inline packages/libs/react-ui/src/styles/vars.css.ts:123]: Are you sure these should be transparent? Normally I would not expect borders to be transparent

**Review by eileenmguo — COMMENTED** [review]
Looking good! Thanks for this refactor 💯

File: packages/libs/react-ui/src/styles/vars.css.ts (hunk 2 above)
Comment (rezaerami) [reply packages/libs/react-ui/src/styles/vars.css.ts:123]: yep following the figma they should have
https://www.figma.com/file/cNQkFOjrqO3PAYv7TSIhpB/Foundation?node-id=122%3A2631&mode=dev

**Review by eileenmguo — APPROVED** [review]

**Review by MRVDH — APPROVED** [review]

**Pull request merged**